src/parser.js

```diff
@@ -92,6 +92,7 @@ export default class Parser extends Stream {
     let currentMap;
     // if specified, the active decryption key
     let key;
+    let hasParts = false;
```

Comment:
Added this to keep renditionReport warnings firing if we get a rendition report before any parts.